src/mlpack/bindings/go/CMakeLists.txt

```diff
@@ -0,0 +1,264 @@
+if (BUILD_GO_BINDINGS)
+  ## We need to check here if Go is even available.  Although actually
+  ## technically, I'm not sure if we even need to know!  For the tests though we
+  ## do.  So it's probably a good idea to check.
```

Comment:
Hmm, this comment is a bit meandering (I have a feeling that I wrote this a while ago... I'm not sure if that's right though... but I think I am to blame...), do you think we should update it to just say `We need to check here if Go is even available, since the tests will need it.`? :)

Reply:
This is fixed now. :+1: